doc/source/conf.py

```diff
@@ -172,7 +172,9 @@
 import pandas  # isort:skip
 
 # version = '%s r%s' % (pandas.__version__, svn_version())
-version = str(pandas.__version__)
+# version = str(pandas.__version__)
+
+version = str(pandas.__version__).split("+")[0]
```

Comment:
Can you revert this change here? (that's from the other PR, because of making this branch after you worked on the other PR from the main branch)